guides/intro_to_git.md

```diff
@@ -0,0 +1,316 @@
+
+5. Click Create Pull Request.
+
+6. Fill out any information related to the Pull request, title, description, any comments. On the rigth hand side you can assign yourself as the Assignee, and assign someone in your team to be the Reviewer. Then, select Create Pull Request.
+
+# General: How to accept a pull request
+
+Once you are assigned the pull request's reviewer, you should receive an email notification from GitHub with a link to the pull request's webpage. You can also view any active pull requests by clicking on Pull Requests on the top bar menu.
+
+![](../images/pull_requests.png)
+
+On the Pull request page, there are four tabs: Conversation, Commits, Checks and Files changed:
+
+### Conversation
+
+**Conversation** is the main pull request page. It contains the pull request's description, timeline and comments left by the assignee or reviewer. To merge a pull request you have 3 presented options:
+
+* Merge pull request: this will keep all commits history of the source branch and will be added to the target branch. Creates a symbolic commit.
+* Squash and merge: ticking this will merge the history of commits for the source branch into one commit. This can be useful when the pull request is not a major update and you don't want your repository's history of commits to contain commits with messages such as "fixed typo", "replaced incorrectly placed image". Creates a symbolic commit.
+* Rebase and merge: adds all source branch commits in front of the target branch's commits. Does not create a symbolic commit.
+
+### Commits
+
+**Commits** tab displays a history of the pull request's commits. So for example, if the reviewer decides that minor adjustments are required and the assignee pushes these changes to Gitlab, then these commits will be captured in this tab.
+
+### Files changes
+
+**Files changed** tab, here you can see all the new changes in the pull request's source branch. You can also opt for a side-by-side comparison, and see line-by-line how the master version and new branch version compare. You can also leave comments on each file's line, which will also be recorded in the Conversation tab's timeline.
+
+### Checks
+
+**Checks** is part of the GitHub Actions functionality, which enchances your code review processes with apps and automated reviewing checks. This is inactive on default.
+
+If you are happy to merge the two branches, then select the type of Pull Request you wish to apply and the pull request should be completed in a few seconds.
+
+# General: How to update your local main branch to the latest remote main version on GitHub/GitLab
+
+1. Open the Anaconda Prompt terminal or Git Bash or any other terminal you wish to use. This can be accessed by typing, for example Anaconda Prompt, in the search bar, after opening the search icon on your Windows toolbar.
```

Comment:
Do we need to repeat this line about terminal usage everywhere? If we do, we should at least be consistent.